src/service/world-api-adapter.js

```diff
@@ -169,7 +171,9 @@ module.exports = function (config) {
         },
 
         /**
-        * Update a World, for example to add the roles to the world.
+        * Updates a World, for example to add the roles to the world.
+        *
+        * Using this method is rare. It is more common to have world configuration done at the project level, rather than at the world level. For example, each world in your project probably has the same roles for end users. And your project is probably either configured so that all end users share the same world (and run), or smaller sets of end users share worlds — but not both.
```

Comment:
Nitpicking, but I'm slightly uncomfortable terming this as 'rare' when our sample-size (so far) is 1 :)

Let's just say "You'd typically configure the.." instead?